src/main/java/edu/kit/hci/soli/domain/NonOidcUser.java

```diff
@@ -0,0 +1,11 @@
+package edu.kit.hci.soli.domain;
+
+import jakarta.persistence.Entity;
+import lombok.Data;
+
+@Entity
+@Data
+public class NonOidcUser extends User{
```

Comment:
Should we really use inheritance here?
Since Spring effectively provides us with an ID that is unique even across services (`"service" + id` if nothing else), we can probably unify our model.